sql/core/src/test/resources/sql-tests/inputs/postgreSQL/join.sql

```diff
@@ -1089,10 +1089,9 @@ order by fault;
 -- left join unnest(v1ys) as u1(u1y) on u1y = v2y;
 
 -- [SPARK-28382] Array Functions: unnest
--- select * from
--- (values (1, array(10,20)), (2, array(20,30))) as v1(v1x,v1ys)
--- left join (values (1, 10), (2, 20)) as v2(v2x,v2y) on v2x = v1x
--- left join unnest(v1ys) as u1(u1y) on u1y = v2y;
+create or replace temporary view v1 as select * from values (1, array(10,20)), (2, array(20,30)) as v1(v1x,v1ys);
+create or replace temporary view v2 as select * from values (1, 10), (2, 20) as v2(v2x,v2y);
```

Comment:
Hm, why is it different from commented? Does it match to PostgreSQL's?

Reply:
we don't support unnest as table here yet

Reply:
I can revert these changes if we shouldn't do that

Reply:
Let's revert. I think the point of point PostgreSQL tests is to support the same syntax.

Reply:
ok